spec/controllers/api/v1/rents_controller_spec.rb

```diff
@@ -1,4 +1,5 @@
 require 'rails_helper'
+require 'sidekiq/testing'
```

Comment:
if you put this in the `rails_helper.rb` you can avoid requiring it in every spec. Just so you know for the future ;)